spec/integration/lib/auth0/api/v2/api_resource_servers_spec.rb

```diff
@@ -22,6 +22,29 @@
     end
   end
 
+  describe '.resource_servers', vcr: true do
+    let(:resource_servers) do
+      client.resource_servers
+    end
+
+    it 'should return at least 1 result' do
+      expect(resource_servers.size).to be > 0
+    end
+
+    it 'should get the test server' do
+      expect(resource_servers).to include(test_server)
+    end
+
+    it 'should return the first page of one result' do
+      results = client.resource_servers(
+        page: 0,
+        per_page: 1
+      )
+      expect(results.first).to equal(results.last)
+      expect(results.first).to eq(test_server)
```

Comment:
This is not necessary, and is failing on my end when I run the integration tests on your branch. We're not pulling by any specific order here so the newest may (or may not) be the first one pulled.

Reply:
I've removed the `expect(results.first).to eq(test_server)` line, which I _believe_ is the only one you objected to. If you want me to remove that whole test about paging, though, I can do that instead.